examples/interactive_play.py

```diff
@@ -69,13 +69,17 @@
     observations_to_image,
     overlay_frame,
 )
+from habitat_baselines.articulated_agent_controllers import (
```

Comment:
This is the only import of habitat-baselines into examples. examples used to work without having to install baselines and this breaks this feature.
How will `articulated_agent_controllers` be used in the future and is it worth putting into habitat-lab ?

Reply:
Yes, I think we could make a case that it should live under lab, specially since some actions will need to have instances of these controllers. I am going to keep it here for this review, and so that it is part of the "fast review process" for now.